mozregression/cli.py

```diff
@@ -386,13 +386,10 @@ def _convert_to_bisect_arg(self, value):
         try:
             value = parse_date(value)
         except DateFormatError:
-            try:
-                new_value = parse_date(date_of_release(value))
-                self.logger.info("Using date %s for release %s"
-                                 % (new_value, value))
-                value = new_value
-            except UnavailableRelease:
-                pass
+            new_value = parse_date(date_of_release(value))
```

Comment:
@wlach https://github.com/mozilla/mozregression/commit/e6c8c3afde4f388d34c1b516db52f51c01ca9f79#diff-a753870fc2c6fa4864480ab7b1c1cd57L479
I checked last commit of these, this function seems to return only datetime.  But it might be hash which leads to match inbound builds.  Therefore we passed UnavailableRelease here.
Since this change broke 8 unittests we should be careful.